src/Tizen.Security.DevicePolicyManager/Tizen.Security.DevicePolicyManager/EmailPolicy.cs

```diff
@@ -0,0 +1,121 @@
+﻿/*
+ *  Copyright (c) 2018 Samsung Electronics Co., Ltd All Rights Reserved
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License
+ */
+
+using System;
+
+namespace Tizen.Security.DevicePolicyManager
+{
+    /// <summary>
+    /// The EmailPolicy provides methods to manage email policies.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class EmailPolicy
```

Comment:
I'll change like below. Is it okay? 

```
class EmailPolicy : IDisposable
{
        private bool _disposed = false;

        ~EmailPolicy()
        {
            this.Dispose(false);
        }

        protected virtual void Dispose(bool disposing)
        {
            if (!_disposed)
            {
                if (disposing)
                {
                    // to be used if there are any other disposable objects
                }

                if (_popImapCallbackId != 0)
                {
                    RemovePopImapPolicyChangedCallback();
                }

                _disposed = true;
            }
       }
        public void Dispose()
        {
            Dispose(true);
            GC.SuppressFinalize(this);
        }
}
```

Reply:
It will cause second issue.
```
var policy = new DevicePolicyManager().GetPolicy<EmailPolicy>();
policy.PopImapPolicyChanged += (s, e) => { // it will be not raised }
policy = null;
System.GC.Collect();
```
when native callback was riased C# event is not raised

Reply:
But, Your solution also acceptable, with remarks about lifetime